tests/light/src/axosyslog_light/fixtures.py

```diff
@@ -60,7 +60,7 @@ def __call__(self, parser, namespace, path, option_string=None):
             raise argparse.ArgumentTypeError("{0} is not a valid directory".format(path))
 
         binary = Path(install_dir, "sbin/syslog-ng")
-        if not binary.exists():
+        if namespace.runner == "local" and not binary.exists():
```

Comment:
I think this depends on the order of setting `--runner` and `--installdir`. The user should not use `--installdir` with `--runner=docker`, so I think this can only happen as  a user error. If we have make targets or such, we should make sure to not include these two options at the same time, I believe.